src/content/packer/product-landing.json

```diff
@@ -5,7 +5,7 @@
 	},
 	"overview": {
 		"heading": "What is Packer?",
-		"body": "Packer is an open source tool that lets you create identical machine images for multiple platforms from a single source template. Packer can create golden images to use in image pipelines.",
+		"body": "Packer is a tool that lets you create identical machine images for multiple platforms from a single source template. Packer can create golden images to use in image pipelines.",
```

Comment:
We can merge this but ideally education would rework this sentence/ approve it.

Reply:
I think we're fine to go ahead here, and have them look at it after the fact.